reconcile/test/test_ocm_upgrade_scheduler.py

```diff
@@ -74,3 +74,65 @@ def test_update_history(self):
             }
         }
         self.assertEqual(expected, history)
+
+
+class TestVersionConditionsMet(TestCase):
```

Comment:
Potentially may be worth considering an additional test-case proving the outcome if upgrade-condition-soak-days == actual-soak-days (exactly equals).

Otherwise I think this looks good.

Reply:
added an equality test. in any case, this case is very unlikely since soak days are a float.